Code/Tools/FBuild/FBuildCore/Graph/ObjectNode.cpp

```diff
@@ -2343,6 +2419,23 @@ bool ObjectNode::CompileHelper::SpawnCompiler( Job * job,
         }
     #endif
 
+    bool isCompilerClangTidy = false;
+    if ( m_HandleOutput )
+    {
+        if (job->IsLocal())
+        {
+            isCompilerClangTidy = compilerNode->GetCompilerFamily() == CompilerNode::CompilerFamily::CLANG_TIDY;
+        }
+        else
+        {
+            const AString& file = job->GetToolManifest()->GetFiles()[0].GetName();
+            if (file.EndsWith("clang-tidy") || file.EndsWith("clang-tidy.exe"))
```

Comment:
This duplicates detection of clang tidy elsewhere and is inconsistent (it's case sensitive here)